documentation/docs/user_guide/inputs/pftlookup_csv.md

```diff
@@ -0,0 +1,128 @@
+# pftlookup.csv
+
+This file is required by CASA-CNP and contains biome (plant function types) specific parameters
+on soil, vegetation carbon and nutrients dynamics.
+
+!!! Note "Parameter names"
+
+     The Variable column shows the parameter names as they appear in the pftlookup.csv table. The CABLE variable is the name of the variable in the CABLE source code. All alternative parameter names in the Description column are from Appendix A of Wang et al. (2010).
+
+| Variable       | <div style="width:250px">CABLE variable</div> | Description           |
+|----------------|-----------------------------------------------|-----------------------|
+| Kroot          | `casabiome%kroot(nv)`                         | Unused \( (m^{-1}) \) |
+| rootdepth      | `casabiome%rootdepth(nv)`                     | Unused \( (m) \) |
+| kuptake        | `casabiome%kuptake(nv)`                       | Unused \( (-) \) |
+| krootlen       | `casabiome%krootlen(nv)`                      | Unused \( (m \cdot gC^{-1}) \) |
+| KminN          | `casabiome%kminN(nv)`                         | $k_{n,up}$ an empirical parameter relating plant nitrogen uptake to soil mineral N amount. Used in `casa_nuptake()` during computation of N uptake by plants (`xnuptake`) and allocation of uptaken N to plants. \( (gN \cdot m^{-2}) \) |
+| Kuplabp        | `casabiome%Kuplabp(nv)`                       | $k_{p,up}$ an empirical parameter relating plant P uptake to labile P pool size in the soil. Used in `casa_puptake()` during calculation of P uptake by plants and the allocation to leaf wood and root. \( (gP \cdot m^{-2}) \)|
+| Fracherb       | `xfherbivore(nv)`                             | Fraction of net foliar production consumed by herbivores. Fraction of the turnover rate that affects senescence: <ul><li>if`Fracherb=1`: senescence is only related to cold and dry stress and there is no turnover factor</li><li> if `Fracherb=0` senescence has an additional turnover rate as set by leaf age. </li></ul>Used to calculate `casabiome%plantrate(nv,leaf)` used in calculation of plant pool senescence rate. Possibly herbivores were removed from CASA when coupled to CABLE. \( (-) \) |
```

Comment:
We should leave this variable in (uncomment the line in casa_inout.F90). The original source for this is: Randerson JT et al. (1996). Global Biogeochemical Cycles, 10(4):585-602.  Table A1.
In the future, this parameter can be adjusted to account for increase insect herbivore under prolong drought.

Reply:
Great, I was just wondering if I had understood it correctly. I will merge it now.